spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
             profile::core::docker
             profile::core::docker::prune
```

Comment:
Replace with `it { is_expected.to contain_cron__job('docker_prune') }`

Reply:
great idea, just test the resource, not the class 👍🏻  changed.